lib/core/util.js

```diff
@@ -200,6 +200,10 @@ function parseHeaders (headers, obj = {}) {
   return obj
 }
 
+function parseRawHeaders (headers) {
+  return headers.map(header => header.toString());
+}
```

Comment:
Do we even need this? I think it's fine to leave this conversion to user space, no? That way we don't have to add another option when someone wants the "raw" headers as buffers.